clang/lib/CodeGen/CMakeLists.txt

```
@@ -1,3 +1,7 @@
+if(EXISTS ${LLVM_MAIN_SRC_DIR}/include/llvm/CodeGen AND CLANG_BUILT_STANDALONE)
```

I think that I would rather have the cases mirrored here - the path should always exist in general, it is only the standalone case that is special.

```suggestion
if(CLANG_BUILT_STANDALONE AND EXISTS ${LLVM_MAIN_SRC_DIR}/include/llvm/CodeGen)
```

---

Ok, will do.